### PR TITLE
Add initial gold standard examples

### PR DESCRIPTION
**Pull request by jessicaaustin**

### CODE REVIEW

File: _docs/ioos-metadata-profile-v1-2.md [hunk 1]
```diff
@@ -51,55 +49,171 @@ summary:  This is the currently active IOOS Metadata Profile version.  See links
...
+platform (global attribute) | ACDD/NCEI |  Name of the *type* of platform(s) that supported the sensor data used to create this data set or product. Platforms can be of any type, including satellite, ship, station, aircraft or other. The controlled vocabulary must be indicated in the `platform_vocabulary` field. <br><br>Example (global variable): {::nomarkdown}<ul> <li> <b><code>platform "buoy";</code></b> <li><b><code>platform_vocabulary "IOOS Platform Vocabulary";</code></b> </ul>{:/} | global | **required**
```
Comment (mwengren) [inline _docs/ioos-metadata-profile-v1-2.md]: @jessicaaustin I noticed a small error in the platform (global) example I made a note to fix. 

Should be:

```suggestion
platform (global attribute) | ACDD/NCEI |  Name of the *type* of platform(s) that supported the sensor data used to create this data set or product. Platforms can be of any type, including satellite, ship, station, aircraft or other. The controlled vocabulary must be indicated in the `platform_vocabulary` field. <br><br>Example (global variable): {::nomarkdown}<ul> <li> <b><code>platform "buoy";</code></b> <li><b><code>platform_vocabulary "https://mmisw.org/ont/ioos/platform";</code></b> </ul>{:/} | global | **required**
```

**Comment by mwengren** [comment]
@jessicaaustin Looks like a great start!  I like the reorganization of the 1.2 guidance into categories.  

I'm wondering what to do about the WMO ID guidance and if we choose to make changes to accommodate the OceanSITES `wmo_platform_code` per Kevin's suggestion, how to incorporate into the guidance, because it would complicate the recommended|required rules significantly.  We can discuss tomorrow.   

Thanks for getting this started.

**Comment by jessicaaustin** [comment]
@mwengren Yeah, the WMO code aspect definitely should be in here. But I think it can be a separate PR? We haven't even nailed down what it should say yet. :)

We should give @kwilcox a chance to review first, but would you be ok with merging this? I think it'll be helpful for our discussion tomorrow.

**Comment by mwengren** [comment]
Happy to merge whenever, it is a big improvement.  I'll let @kwilcox review first, and I'm actually not sure what happens in this case, but did you see my comment above on a code example error?  Do I need to change that before I merge, or will it be automatically merged.  Never used the code suggestion feature before.

**Comment by mwengren** [comment]
@jessicaaustin I also noticed item 5 and 6 in the caveats are redundant now (I had added 6 a few weeks ago when I saw you started working on them).  Can you remove that in your branch before merge?

File: _docs/ioos-metadata-profile-v1-2.md (hunk 1 above)
Comment (kwilcox) [reply _docs/ioos-metadata-profile-v1-2.md]: Fixed

**Comment by kwilcox** [comment]
> @jessicaaustin I also noticed item 5 and 6 in the caveats are redundant now (I had added 6 a few weeks ago when I saw you started working on them). Can you remove that in your branch before merge?

Fixed (removed 5)

**Comment by mwengren** [comment]
@kwilcox I think maybe we want 5 instead of 6?  IDK.  I can go ahead and merge and we can fix again later if necessary.

**Pull request merged**